.github/CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+# Code owners are the default owners for everything in
+# this repository. The owners listed below will be requested for
+# review when a pull request is opened.
+*       @global-owner1 @global-owner2
```

Comment:
@misscoded ahhh yep, looks like it's throwing an error when having invalid usernames here 😢

Reply:
I updated the template again to keep in the commented line and added some additional instructions to help out with next steps for adding CODEOWNERS, let me know your thoughts! 🙌